Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "fil_builtin_actors_bundle"
 description = "Bundle of FVM-compatible Wasm bytecode for Filecoin builtin actors"
-version = "9.0.1"
+version = "9.0.2"
 license = "MIT OR Apache-2.0"
 authors = ["Protocol Labs", "Filecoin Core Devs"]
 edition = "2021"
```

Comment:
Why aren't the `[target.'cfg(target_arch = "wasm32")'.dependencies]` dependencies getting bumped?